packages/@o3r/analytics/schematics/index.it.spec.ts

```diff
@@ -54,6 +55,12 @@ describe('new otter application with analytics', () => {
       );
       addImportToAppModule(appFolderPath, 'TestComponentContModule', 'projects/test-app/src/components/test-component');
 
+      const diff = getGitDiff(execAppOptions.cwd as string);
```

Comment:
This is a bit confusing because rely on the time the commit has been done (and the file it contains).
Maybe it would make more sense to have this diff in output of the `packageManagerExec` directly?

Reply:
the only commit done is at the end of the prepareTestEnv, so the gitDiff contains only what have been changed inside the test itself
I want to test that at the end of the test, the project `dont-modify-me` which is the project[0] hasn't been modified, and I didn't want to test it after each and every command

Reply:
But we have a task to create a commit after the `ng add @o3r/core`, this may be the case of several rules actually.
I suggest to provide a return of `packageManagerExec` to not be dependent of the task implementation

Reply:
I've added a git tag after the initialization of the test environment to handle the case where commits are added during the test